package.json

```diff
@@ -23,37 +23,32 @@
     "node": ">=20"
```

Comment:
Seems like lower node versions (tested on 18.12.1) and minor 20 versions (tested on 20.6.1) throw error
`TypeError: The "path" argument must be of type string. Received undefined`
when running tests.

This however is not an issue on 20 lts (20.11.1) and  onwards.

Should we specify just to make sure ? 

`"node": ">=20.11.1"`

Most likely people tend to install lts version of major required ones but do we want to make sure ?